src/utils/format-numbers.js

```diff
@@ -92,6 +93,14 @@ export function formatMilliseconds(
   return formatNumber(time, significantDigits, maxFractionalDigits) + 'ms';
 }
 
+export function formatSeconds(
+  time: Seconds,
+  significantDigits: number = 2,
+  maxFractionalDigits: number = 3
+) {
+  return formatNumber((time / 1000), significantDigits, maxFractionalDigits) + 's';
```

Comment:
eslint/prettier seems to whine about this line, you need to run `yarn lint-fix` :) (or better: configure your editor to run `eslint --fix` for you on save).